### ISSUE TITLE
Possible to pull the names of the 'sections' of a youtube video?

### ISSUE THREAD

**Issue by DrAutissimo** [ytdl-org/youtube-dl#25649]
## Checklist

- [x] I'm asking a question
- [x] I've looked through the README and FAQ for similar questions
- [x] I've searched the bugtracker for similar questions including closed ones

## Question

Hi to you all,

I recently found out that the (relatively) new chapters, that creators can put in by putting timestamps and a string in the comments/description, actually get written as chapters upon downloading.
( https://www.youtube.com/watch?v=HZTStHzWRxM This video has them, and explains them a bit.)

Now, I wanted to know if it is somehow possible to treat the seperate chapters as seperate files, give them their own metadata, and then merge them in a container?
Like, I download a long collection of music where the individual tracks are given as chapters.
If I open the resulting single file with Foobar, the only metadata I can manage to get is that the 'Title' just is the file/videoname, and the 'Artist' is the uploaders name, but each chapter is its own track, so I would like the title of each chapter to be the name of it. Is this somehow possible?
Also, does YTDL only get the chapters from the description, or does it also work if it is f.e. in a pinned comment?

If this specific question has been asked before, I am again, really sorry.

Thanks for any answers in advance.

**Comment by DrAutissimo** [comment]
I should've removed the open tag I guess.
I am just very very stupid.
Sorry.

**Comment by DrAutissimo** [comment]
Ok, I tried to figure this out on my own, since the functionality apparently is here for a while now, with similar questions going on, but these mostly try to split the file, instead of changing the metadata inside the container.
(#24858 #25005 etc.)

I edited my original question, so that the already answered questions are not there anymore, but my case still remains, maybe this is not possible due to the way container formats work, in that case sorry.
I can't even figure out how to f.e. write the chapter names and timestamps into a file, without just copy-pasting the description...
